### DISCUSSION TITLE
Interprocess solution

### DISCUSSION THREAD

**Discussion by xuyang-nudt** [bitwuzla/bitwuzla#70] [General] [answered]
How to obtain the solution in the bitwuzla solution process?

**Comment by mpreiner** [comment, marked as answer]
If you use Bitwuzla via the API you can use [Bitwuzla::get_value()](https://bitwuzla.github.io/docs/cpp/classes/bitwuzla.html#_CPPv4N8bitwuzla8Bitwuzla9get_valueERK4Term) (C++ API), [bitwuzla_get_value()](https://bitwuzla.github.io/docs/c/types/bitwuzla.html#_CPPv418bitwuzla_get_valueP8Bitwuzla12BitwuzlaTerm) (C API) or [Bitwuzla.get_value()](https://bitwuzla.github.io/docs/python/interface.html#bitwuzla.Bitwuzla.get_value) (Python API). If you use the SMT-LIBv2 interface, you can get a model with the `(get-model)` command or ask values for specific terms via `(get-value)`. In all of these cases, you have to enable the model generation option.
